volontulo/templates/homepage.html

```diff
@@ -49,14 +34,44 @@
                             </div>
                         </div>
                         <h3><a class="text-warning" href="{% url 'offers_view' o.title|slugify o.id %}">{{ o.title }}</a></h3>
-                        <div class="text-right">Organizator:<a class="text-warning" href="{% url 'organization_view' slug=o.organization.name|slugify id_=o.organization.id %}">{{ o.organization.name }}</a></div>
+                        <div class="text-right">Organizator: <a class="text-warning" href="{% url 'organization_view' slug=o.organization.name|slugify id_=o.organization.id %}">{{ o.organization.name }}</a></div>
                     </div>
                 </div>
             </div>
         {% endfor %}
         </div>
     {% else %}
-        <p>Brak ofert spełniających podane kryteria</p>
+        <p class="lead">Zgłoś swoją ofertę aby znaleźć wolontariuszy do pomocy w akcji!</p>
+        {% if user.is_anonymous and not user.is_authenticated %}
+        <div class="media">
+            <div class="media-left media-middle">
```

Comment:
that's how bootstrap works
media is ectually a table and it has left body and right cells
their content can be top (default) middle and bottom aligned
but you can't tell all cells to middle align